tests/app/core/models/session-model.spec.ts

```diff
@@ -61,7 +49,7 @@ describe('Session Model', () => {
     });
 
     it('should throw error on session data with undefined repo for open phase', () => {
-      const modifiedSessionData: SessionData = { ...validSessionData, openPhases: [Phase.phaseBugReporting] };
+      const modifiedSessionData: SessionData = { ...MODERATION_PHASE_SESSION_DATA, openPhases: [Phase.phaseBugReporting] };
       of({ ...modifiedSessionData, phaseBugReporting: undefined })
```

Comment:
Maybe we can combine the overrides in the `modifiedSessionData` instead of doing some there and some in the `of()` ?

Reply:
An easier way, if we are using `MODERATION_PHASE_SESSION_DATA` as the default pre-override, is we could just set `phaseModeration: undefined` in line 52.

Reply:
I think it would be better just to standardize the tests formats for easier reading! Instead of overriding some here and some there ><

Reply:
Ahh that makes sense, in that case we can just have Line 53 as `of({ ...MODERATION_PHASE_SESSION_DATA, phaseModeration: undefined })` cuz changing the openPhases to bugReporting then setting the bugReporting repo to undefined is abit like double work? haha 😅

Reply:
Yea truth that.. alright!